test/libsolidity/smtCheckerTests/loops/while_loop_array_assignment_storage_storage.sol

```diff
@@ -23,4 +23,3 @@ contract LoopFor2 {
 // ----
 // Warning 6328: (320-339): CHC: Assertion violation happens here.
 // Warning 6328: (343-362): CHC: Assertion violation happens here.
-// Warning 4661: (296-316): BMC: Assertion violation happens here.
```

Comment:
What made it not asserting anymore?

Reply:
The first assertion is actually safe, so ideally this is the expected result. Sometimes if the encoding changes ever so slightly the result might also change, in which case CHC wasn't (before) able to solve this, and BMC reports it as a false positive.

Reply:
Maybe than deleting comment (about false positive) just before first assertion?

Reply:
Ok, will delete.